api/controllers/InviteController.js

```diff
@@ -16,8 +16,8 @@ module.exports = {
     req.session.destroy();
 
     // Get full URL
-    const protocol = req.connection.encrypted? 'https' : 'http';
-    const medicalProfessionalRoute = sails.config.routes['GET /en/medical-professional'].i18n[req.i18n.locale];
+    const protocol = req.connection.encrypted ? 'https' : 'http';
+    const medicalProfessionalRoute = sails.route('medical-professional');
     let medicalProfessionalUrl = protocol + '://' + req.headers.host + medicalProfessionalRoute;
```

Comment:
Suggestion for part for removing placeholder for valid date.

```suggestion
    let medicalProfessionalUrl = protocol + '://' + req.headers.host + medicalProfessionalRoute;
    
    // Get timeframe for when the consent expires
    let submissionMoment = moment(medicalReport.applicantSubmittedAt);
    let validTil = submissionMoment.add(3, 'y').format('LL');
```

Reply:
You'll also need to add to the top:

`const moment = require('moment');` 

and inject into the view data dictionary.

Reply:
Merci, I added all of this!